### PR TITLE
Fix typos in locales and format the yml files

### PR DESCRIPTION
**Pull request by vassyz**
Found a few typos in the locales and my editor also formatted a few files.

### CODE REVIEW

File: config/locales/support_interface/support_interface.yml [hunk 1]
```diff
@@ -223,10 +223,9 @@ en:
           attributes:
             grade:
               blank: Enter the grade
+              invalid: Enter a valid grade
```
Comment (tomas-stefano) [inline config/locales/support_interface/support_interface.yml:226]: How was before this was added?

**Review by tomas-stefano — APPROVED** [review]
Looks good, after @MaeveRoseveare approved, feel free to merge :)

**Comment by vassyz** [comment]
@MaeveRoseveare do you need a review app for this? It's pretty straightforward if you look at the changes.

File: config/locales/support_interface/support_interface.yml (hunk 1 above)
Comment (tomas-stefano) [reply config/locales/support_interface/support_interface.yml:226]: Oh below. I see.

**Pull request merged**